tests/network/l2_bridge/conftest.py

```diff
@@ -288,11 +294,12 @@ def bridge_attached_vm(
 
 
 @pytest.fixture(scope="class")
-def l2_bridge_running_vm_a(namespace, worker_node1, l2_bridge_all_nads, unprivileged_client):
+def l2_bridge_running_vm_a(namespace, worker_node1, dhcp_nad, unprivileged_client, l2_bridge_running_vm_b):
```

Comment:
Removing `l2_bridge_all_nads` is not possible, it is used below (line 320). You can only add.